apps/bluebomber/bluebomber.star

```diff
@@ -0,0 +1,305 @@
+
+BOSS_LIST = {
+    "MEGA": "MEGA",
+    "CUT": "CUT",
+    "GUTS": "GUTS",
+    "ICE": "ICE",
+    "FIRE": "FIRE",
+    "BOMB": "BOMB",
+    "ELEC": "ELEC",
+    "CRASH": "CRASH",
+    "BUBBLE": "BUBBLE",
+    "METAL": "METAL",
+    "AIR": "AIR",
+    "QUICK": "QUICK",
+    "FLASH": "FLASH",
+    "WOOD": "WOOD",
+    "HEAT": "HEAT",
+    "NEEDLE": "NEEDLE",
+    "MAGNET": "MAGNET",
+    "GEMINI": "GEMINI",
+    "HARD": "HARD",
+    "TOP": "TOP",
+    "SNAKE": "SNAKE",
+    "SHADOW": "SHADOW",
+    "SPARK": "SPARK",
+    "TOAD": "TOAD",
+    "BRIGHT": "BRIGHT",
+    "PHARAOH": "PHARAOH",
+    "RING": "RING",
+    "DUST": "DUST",
+    "SKULL": "SKULL",
+    "DIVE": "DIVE",
+    "DRILL": "DRILL",
+    "GRAVITY": "GRAVITY",
+    "GYRO": "GYRO",
+    "CRYSTAL": "CRYSTAL",
+    "NAPALM": "NAPALM",
+    "STONE": "STONE",
+    "CHARGE": "CHARGE",
+    "WAVE": "WAVE",
+    "STAR": "STAR",
+    "BLIZZARD": "BLIZZARD",
+    "WIND": "WIND",
+    "FLAME": "FLAME",
+    "PLANT": "PLANT",
+    "TOMAHAWK": "TOMAHAWK",
+    "YAMATO": "YAMATO",
+    "KNIGHT": "KNIGHT",
+    "CENTAUR": "CENTAUR",
+    "CONCRETE": "CONCRETE",
+    "JEWEL": "JEWEL",
+    "HORNET": "HORNET",
+    "TORNADO": "TORNADO",
+    "PLUG": "PLUG",
+    "GALAXY": "GALAXY",
+    "MAGMA": "MAGMA",
+    "SPLASH": "SPLASH",
+    "CHILL": "CHILL",
+    "BLADE": "BLADE",
+    "SOLAR": "SOLAR",
+    "SHEEP": "SHEEP",
+    "COMMANDO": "COMMANDO",
+    "NITRO": "NITRO",
+    "STRIKE": "STRIKE",
+    "PUMP": "PUMP",
+}
+
+def main(config):
```

Comment:
In order to make this usable to folks in the Tidbyt mobile app, we would need a `Dropdown` schema field to be able to select which boss and gender to display: https://tidbyt.dev/docs/reference/schema.

Reply:
Good ideas! It wasn't really the intent of the app, however, it was just meant to be a random one displayed on each load (similar to the Nyan Cat app). If you think that'd be a good option for mobile users then I'm happy to add it! Gender isn't really a configurable option, as it's just there to catch the one case of "SPLASH WOMAN" being displayed.

There's I believe 64 boss options so do you feel a dropdown will be workable with that amount of choices? Not sure of a better option however.

Reply:
Oh! My bad. I completely missed the randomized aspect of this change.